backend/app/api.py

```diff
@@ -28,17 +28,17 @@
 )
 
 
-@app_obj.on_event("startup")
-def on_startup():
-    model.create_db_and_tables()
+# @app_obj.on_event("startup")
+# def on_startup():
+#     model_helpers.create_db_and_tables()
 
 # route handlers
 
 
 @app_obj.post("/token")
 async def login(
         form_data: Annotated[OAuth2PasswordRequestForm, Depends()],
-        session: Session = Depends(model.get_db_session)):
+        session: Session = Depends(model_helpers.get_db_session)):
```

Comment:
this had to be moved out of the model module to be able to do automatic migrations